### ISSUE TITLE
About the evaluation metric

### ISSUE THREAD

**Issue by Zhong1015** [ibrahimethemhamamci/CT2Rep#25]
Dear authors, thank you for your exciting work.
Recently I want to reproduce the work, I found that training loss and validation loss reduced but the Bleu do not change after the first epoch. 
As I did not find the code would just output the best Bleu, so I am a little confused with the result, I would like to know if there are any problem, the result is:
Train loss:0.7575
Val Bleu1: 0.4500
Val Bleu2: 0.3663
Val Bleu3:0.3157
Val Bleu4: 0.2827
Val_loss:0.7342
Thank you again for your effort and I am looking forward to your response !
